index.coffee

```diff
@@ -21,6 +21,8 @@ RegularExpressionLiteral = ///
     (?![ / \] \\ ]).
```

Comment:
Would it make more sense to remove `\]` from here instead?

Reply:
Right, fixed.